core/src/services/routing.js

```diff
@@ -640,6 +640,9 @@ class RoutingClass {
   }
 
   appendModalDataToUrl(modalPath, modalParams) {
+    if (sessionStorage.getItem('historyState')) {
```

Comment:
can you please add a one line comment here as to why we check for this historyState, what it means and why we need to return if it exists. I cannot understand. I guess I would forget in a month or two.

Reply:
Good point.. was wrong anyway..